docs/user/security/authorization/index.asciidoc

```diff
@@ -17,10 +17,60 @@ To create a role that grants {kib} privileges, open the menu, then click *Stack
 [float]
 ==== Required permissions
 
-The `manage_security` cluster privilege is required to access role management.
+The `manage_security` {ref}/security-privileges.html#privileges-list-cluster[cluster privilege] is required to access role management.
+
+[[adding_cluster_privileges]]
+==== Cluster privileges
+
+Cluster privileges grant access to monitoring and management features within {es}. They also enable certain <<management, Stack Management>> capabilities within {kib}.
+
+Refer to {ref}/security-privileges.html#privileges-list-cluster[cluster privileges] for a complete description of available options.
+
+[[adding_index_privileges]]
+==== Index privileges
+
+Each role can grant access to multiple data indices, and each index can have a different set of privileges.
+We recommend granting the `read` and `view_index_metadata` privileges to each index that you expect your users to work with inside of {kib}.
+
+Refer to {ref}/security-privileges.html#privileges-list-indices[index privileges] for a complete description of available options.
+
+Document-level and field-level security affords you even more granularity when it comes to granting access to your data.
+With document-level security ("DLS"), you can write an {es} query to describe which documents this role should grant access to.
+With field-level security ("FLS"), you can instruct {es} to grant or deny access to specific fields within each document.
+
+[[index_privilege_example_1]]
+===== Example 1: Grant access to indices matching the `filebeat-*` pattern
+
+. Enter `filebeat-*` in the **Index** field under the **Index Privileges** section.
+. Enter `read` and `view_index_metadata` in the **Privileges** field under the **Index Privileges** section.
+
+[role="screenshot"]
+image::user/security/images/create-role-index-example.png[Create role with index privileges]
+
+[[index_privilege_dls_example]]
+===== Example 2: Grant read access to specific documents matching the `filebeat-*` pattern
+
+. Enter `filebeat-*` in the **Indices** field under the **Index Privileges** section.
```

Comment:
Same comment as above

1. Go to **Stack Management > Roles**, and then click **Create role**. 
2. In **Index privileges**,  enter:
   `filebeat-*` in the **Index** field
    `read` and `view_index` in the **Privileges** field